docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -112,7 +112,7 @@ called from Node.js callbacks.
 
 #### Using Test Controller Outside of Test Code
 
-You may sometimes need to call the test API from outside the test code. For instance, your [page model](../recipes/using-page-model.md)
+You may sometimes need to call the test API from outside the test code. For instance, your [page model](../recipes/extract-reusable-test-code/use-page-model.md)
```

Comment:
There may be times when you need to call the test API from outside the test code. For instance, your [page model](../recipes/extract-reusable-test-code/use-page-model.md)